konrad/core.py

```diff
@@ -343,5 +368,59 @@ def run(self):
             # Otherweise increase the iteration count and go on.
             else:
                 self.niter += 1
+                self.time += self.timestep
         else:
             logger.info('Stopped after maximum number of iterations.')
```

Comment:
The stop criteria is based on the maximum duration rather than number of iterations, so the logging info would be better if it reflected this :)

Reply:
Fair point. I will update the log message.